controllers/autohealconfig_controller.go

```diff
@@ -47,16 +67,377 @@ type AutohealConfigReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.12.1/pkg/reconcile
 func (r *AutohealConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	autoheal := &v1alpha1.AutohealConfig{}
+	err := r.Client.Get(ctx, req.NamespacedName, autoheal)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			log.Infof("Autoheal CR not found%s", err)
+			return ctrl.Result{}, nil
+		}
+		log.Errorf("Error getting Autoheal CR %s", err)
+		return ctrl.Result{}, err
+	}
+	log.Info("Reconcile")
```

Comment:
```suggestion
```
Just wondering if this logline is needed.

Reply:
sorry yes good catch it was used for testing reconcile times ill remove it